src/resize.cc

```diff
@@ -679,6 +684,15 @@ class ResizeWorker : public NanAsyncWorker {
           return Error(baton, hook);
         }
         baton->outputFormat = "tiff";
+      } else if (outputDzi || (matchInput && inputImageType == ImageType::DZI)) {
+#if (VIPS_MAJOR_VERSION >= 7 && VIPS_MINOR_VERSION >= 41)
```

Comment:
sharp currently requires libvips 7.40.0+, which I believe provides all the API needed for `vips_dzsave`, so this `#if` directive can probably be removed.